### DISCUSSION TITLE
Service "Bing AI copilot"

### DISCUSSION THREAD

**Discussion by pacihak** [ferdium/ferdium-app#1097] [Show and tell]
This is my working custom build, you can add it to regular services.

Name: **Bing AI copilot**
Custom server: **https://www.bing.com/search?q=Bing+AI&showconv=1&FORM=hpcodx**
Custom icon: **https://play-lh.googleusercontent.com/kmxAt02yG8Hp-TzZHDoRGFtB70QFz7njpWQzQp8KANztd-DozlJipu8tprSn80Q8pw=w240-h480-rw**
User Agent: **Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36 Edg/110.0.1587.57**

**Comment by vraravam** [comment]
hi @pacihak - please do not create discussion threads for new service requests. Closing.
